src/ssl.c

```diff
@@ -47478,6 +48136,68 @@ int oid2nid(word32 oid, int grp)
             }
             break;
 
+
+    #ifdef HAVE_ECC
+        case oidCurveType:
+            switch (oid) {
```

Comment:
This looks like a copy of EccNumToNID()???

Reply:
Its similar. `EccEnumToNID` converts our internal curve ID's (`enum ecc_curve_id` found in `ecc.h`) to NID's while this converts the Object ID's (`enum Ecc_Sum` found in `asn_public.h`) to NID's. Our ID's and the Object ID's don't have the same values so `EccEnumToNID` can't be used.